package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nest-starter",
-  "version": "1.0.0",
+  "version": "2.1.0",
   "description": "Nest.js boilerplate for minimalists",
```

Comment:
Please change this to "CarmaTech API based on NestJS framework".
And change the author to you and me as well.